webapp/src/app/services/materials.service.ts

```
@@ -73,26 +77,31 @@ export class MaterilasService {
   }
 
   logMaterial(courseId: string, materialId: string): Observable<any> {
-    return this.http.get<CreateMaterial>(`${this.API_URL}/courses/${courseId}/materials/${materialId}`)
+    return this.http.get<CreateMaterial>(
+      `${this.API_URL}/courses/${courseId}/materials/${materialId}`
+    );
   }
 
   uploadFile(formData: any, materialType: string = 'pdf'): any {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)